pontoon/base/static/js/locale_project.js

```diff
@@ -0,0 +1,65 @@
+$(function() {
+
+  // Search entire project with Enter (and remove default search handler)
+  $('.part .menu .search-project input[type=search]').off('keyup.search').keydown(function (e) {
```

Comment:
Instead of using `off` to un-bind the older event handler, you should instead fix the selector for the other event handler such that this element doesn't get bound in the first place. Otherwise the programmer has to keep in their head every single event handler in the site in order to figure out what events are bound to which elements.